src/main/osd/osd_elements.c

```diff
@@ -1375,12 +1391,16 @@ void osdAnalyzeActiveElements(void)
     }
 #endif // GPS
 
-#ifdef USE_ESC_SENSOR
+#if defined(USE_ESC_SENSOR) || defined(USE_RPM_FILTER)
+    if ((featureIsEnabled(FEATURE_ESC_SENSOR)) || (motorConfig()->dev.useDshotTelemetry)) {
```

Comment:
Has this been tested? Initialisation order is important here, and if `motorConfig()->dev.useDshotTelemetry)` is only set during initialisation _after_ the OSD was initialised, this will not show up.

Reply:
Yes tested on two quads with different FC and esc: one with esc telemetry and one without.
Quad 1: omnibusf4v6 & typhoon 32
Quad 2: omnibusf4sd & sunrise siskin 31a

I tested without props and basic stick moments.  Esc telemetry reading is consistent with bidirectional dshot.  I will some flight tests hopefully over the weekend.